src/object/obj_tx.c

```diff
@@ -917,7 +917,7 @@ dc_tx_commit_cb(tse_task_t *task, void *data)
 	}
 
 	/* Need to refresh the local pool map. */
-	if (tx->tx_pm_ver < oco->oco_map_version) {
+	if (tx->tx_pm_ver < oco->oco_map_version || obj_retry_error(rc)) {
```

Comment:
obj_retry_error() contains many cases, for some special cases, such as "err == -DER_INPROGRESS" or "err == -DER_CSUM" or "err == -DER_TX_BUSY", we do NOT need to refresh the pool map, right?